servicetalk-http-api/src/main/java/io/servicetalk/http/api/SingleAddressHttpClientBuilder.java

```diff
@@ -164,6 +170,12 @@
     @Override
     public abstract SingleAddressHttpClientBuilder<U, R> ioExecutor(IoExecutor ioExecutor);
 
+    @Override
+    public SingleAddressHttpClientBuilder<U, R> executor(Executor executor) {
+        LOGGER.warn("Unimplemented method SingleAddressHttpClientBuilder::executor() called");
```

Comment:
Alternative for logging will be to throw `UnsupportedOperationException`. That way, if users have a custom implementation of the builder, they won't miss this warning after migration to the new method.

Reply:
I considered that but wanted to "do no harm". If they do use the new `executor(Executor)` method though then perhaps it should fail explicitly rather than implicitly fail.

Reply:
I am going to switch to `UnsupportedOperationException`.